src/vs/workbench/api/common/extHostTreeViews.ts

```diff
@@ -201,7 +201,10 @@ export class ExtHostTreeView<T> extends Disposable {
 	constructor(private viewId: string, options: vscode.TreeViewOptions2<T>, private proxy: MainThreadTreeViewsShape, private commands: CommandsConverter, private logService: ILogService, private extension: IExtensionDescription) {
 		super();
 		this.dataProvider = options.treeDataProvider;
-		this.proxy.$registerTreeViewDataProvider(viewId, { showCollapseAll: !!options.showCollapseAll, canSelectMany: !!options.canSelectMany });
+		// {{SQL CARBON EDIT}}
```

Comment:
Where is this failing? I'm assuming its from one of our tests? Why are we not passing in mocked objects?

Reply:
I was manually testing the external table wizard with the latest insiders build.